README.md

```diff
@@ -23,15 +25,37 @@ Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master bra
 
 Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
 
-## Usage
+### Usage for combined policy statements
+
+```hcl
+// TODO: make sure these references to module names are correct once pushed to the registry
+module "ou_scp" {
+  source = "trussworks/ou-scp"
+  target = aws_organizations_organizational_unit.root.id
+
+  # true means the policy is in effect
+  deny_root_account = true
+  deny_leaving_orgs = true
+  deny_creating_iam_users = true
+  deny_deleting_kms_keys = true
+  deny_deleting_route53_zones = true
+  require_s3_encryption = true
+  deny_deleting_cloudwatch_logs = true
+  protect_s3_buckets = true
+  protect_iam_roles = true
+  limit_regions = true
```

Comment:
If you have "true" set you need to provide some regions to limit it too, same for the IAM and S3 bucket protections. If you're going to have an example like this you should add those.

Reply:
😬  idk how I missed those...